src/components/EventListPreview/index.js

```diff
@@ -0,0 +1,21 @@
+import StyledListItem from "../StyledListItem";
+import StyledDivider from "../StyledDivider";
+import shortenText from "../../utils/shortenText";
+
+export default function EventListPreview({ event }) {
+  return (
+    <>
+      <StyledListItem>
+        <time
+          dateTime={event.startDate}
+          aria-label="date"
+        >{`${event.startDate.slice(8, 10)}.${event.startDate.slice(
+          5,
+          7
+        )}.${event.startDate.slice(0, 4)}`}</time>
```

Comment:
Oh one more thing: Please use `Intl.DateTimeFormat` to format date strings instead.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat